### ISSUE TITLE
Is there a license?

### ISSUE THREAD

**Issue by carrotcreamsoup** [jorio/Pomme#1]
Hi,

I was interested in using a portion of this library for my own project, which is under GPL GNUv3. I was wondering if there was a license — or if adding a license would be a possibility — to make use of portions of Pomme?

The module I am interested in is the [mu-law and a-law decoder](https://github.com/jorio/Pomme/blob/master/src/Sound/xlaw.cpp).

Thank you very much!

**Comment by jorio** [comment]
Hi! Good point, I just added [licensing info](https://github.com/jorio/Pomme/blob/master/LICENSE.md).

Most of the library is covered by the MIT license, except for some portions that are LGPL v2.1 because they constitute derivative works from ffmpeg's source code, namely (at the time I'm writing this): IMA4.cpp, MACE.cpp, Cinepak.cpp.

As for xlaw.cpp, although I did generate the conversion tables *using* ffmpeg, that particular source file itself is an original work, so it's MIT. Go ahead and use it!
